torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2700,6 +2700,37 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_fmod_remainder(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = (
+        ((S, S, S), (), 1.5, False),
```

Comment:
I believe 1.5 was added previously to make sure there is no zero in the input. We can drop the `add_other` i.e. 1.5 as we have `exclude_values` to deal with this.

Reply:
Done! Thanks for pointing this out, @kshitij12345!